Eplant/Eplant.tsx

```diff
@@ -28,6 +30,14 @@ export default function Eplant() {
  * The main Eplant component. This is the root of the application. It contains the left nav and the layout.
  * @returns {JSX.Element} The rendered Eplant component
  */
+
+// Setting the sidebar collapse state with jotai
+export const isCollapse = atom(false)
```

Comment:
Can you put this in `Eplant/state/index.tsx`. You also don't need to define a separate setter atom. You should just define a custom hook called useSidebarState (or something along those lines) and use this whenever the collapse state needs to be read or written to. 

See [the use darkMode hook](https://github.com/BioAnalyticResource/ePlant/blob/98fc1a21b622a2bad6af33f95d19810e87142c22/Eplant/state/index.tsx#L210) for how to create a custom hook (just use a regular atom like you have instead of "atomWithOptionalStorage") and see [the darkMode toggle](https://github.com/BioAnalyticResource/ePlant/blob/98fc1a21b622a2bad6af33f95d19810e87142c22/Eplant/UI/LeftNav/index.tsx#L29) for how to use it.

In the above example a separate setter atom is defined but it is unnecessary for the sidebar state.